t/api/14-plugin_obs_rsync_latest_test.t

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Mojo::Base -strict;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use Test::More;
+use Test::Mojo;
+# use OpenQA::Test::Database;
+use OpenQA::Test::Case;
+use Mojo::File qw(tempdir path);
+use File::Copy::Recursive 'dircopy';
+
+my $schema = OpenQA::Test::Case->new->init_data;
+
+$ENV{OPENQA_CONFIG} = my $tempdir = tempdir;
+my $home_template = path(__FILE__)->dirname->dirname->child('data', 'openqa-trigger-from-obs');
+my $home          = "$tempdir/openqa-trigger-from-obs";
+dircopy($home_template, $home);
+$tempdir->child('openqa.ini')->spurt(<<"EOF");
+[global]
+plugins=ObsRsync
+[obs_rsync]
+home=$home
+EOF
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+
+# Allow Devel::Cover to collect stats for background jobs
+$t->app->minion->on(
+    worker => sub {
+        my ($minion, $worker) = @_;
+        $worker->on(
+            dequeue => sub {
+                my ($worker, $job) = @_;
+                $job->on(cleanup => sub { Devel::Cover::report() if Devel::Cover->can('report') });
+            });
+    });
+
+my $app = $t->app;
+$t->ua(OpenQA::Client->new(apikey => 'ARTHURKEY01', apisecret => 'EXCALIBUR')->ioloop(Mojo::IOLoop->singleton));
+$t->app($app);
+my $helper = $t->app->obs_rsync;
+
+subtest 'latest_test' => sub {
```

Comment:
So this looks like all the lines in before are boiler-plate init and common with other test modules, right? Can you just include the following test lines in another test module?